src/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -2,25 +2,59 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Security;
 using Xunit;
 
 namespace System.IO.Tests
 {
     public class EncryptDecrypt : FileSystemTest
     {
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetcoreCoreRT)]
```

Comment:
Now PNSE throw also if path is null. Moved test to Windows only...added PNSE on null param below.